### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by helloyako** [naver/android-imagecropview#19]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'android.graphics.Bitmap com.naver.android.helloyako.imagecrop.view.a.a.a()' on a null object reference	at com.naver.android.helloyako.imagecrop.view.ImageCropViewBase.android.graphics.Bitmap getViewBitmap()(ImageCropViewBase.java:981)
	at com.naver.android.helloyako.imagecrop.view.ImageCropViewBase.com.naver.android.helloyako.imagecrop.model.CropInfo getCropInfo()(ImageCropViewBase.java:973)
	at com.naver.android.pholar.edit.PhotoEditActivity.void onClickCropSelectButton(android.view.View)(PhotoEditActivity.java:1331)
	at com.naver.android.pholar.edit.PhotoEditActivity$1.void onClick(android.view.View)(PhotoEditActivity.java:244)
	at android.view.View.null performClick(null)(View.java:5697)
	at android.widget.TextView.null performClick(null)(TextView.java:10826)
	at android.view.View$PerformClick.null run(null)(View.java:22526)
	at android.os.Handler.null handleCallback(null)(Handler.java:739)
	at android.os.Handler.null dispatchMessage(null)(Handler.java:95)
	at android.os.Looper.null loop(null)(Looper.java:158)
	at android.app.ActivityThread.null main(null)(ActivityThread.java:7225)
	at java.lang.reflect.Method.null invoke(null)(Method.java)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.null run(null)(ZygoteInit.java:1230)
	at com.android.internal.os.ZygoteInit.null main(null)(ZygoteInit.java:1120)
```

**Comment by helloyako** [comment]
add null check code c393e73c74387ccef73e4702ddfa2afc35279326

**Comment by helloyako** [comment]
resolved v1.2.0
